src/elgato/elgato.py

```diff
@@ -26,6 +26,8 @@
 if TYPE_CHECKING:
     from collections.abc import Callable, Coroutine
 
+VERSION = metadata.version(__package__)
```

Comment:
Let's remove the version altogether. It is a local device, and it doesn't matter.